src/datetime.js

```diff
@@ -1081,7 +1081,7 @@ export default class DateTime {
    */
   get isInDST() {
     return this.c
-      ? this.offset > this.set({ month: 1 }).offset || this.offset > this.set({ month: 5 }).offset
+      ? this.offset > this.set({ month: 12 }).offset || this.offset > this.set({ month: 6 }).offset
```

Comment:
Found a bug here?

Reply:
Nope. That should have been a different PR.

This is to make this code similar to what is done in `Info.hasDST()`